.pre-commit-config.yaml

```diff
@@ -57,7 +57,7 @@ repos:
         files: ^tests/app/
 
   - repo: https://github.com/ambv/black
-    rev: 23.10.1
+    rev: 23.11.0
```

Comment:
If python 3.10 is the lower bound of the supported languages, it might be worth setting a 3.10 `language_version` for the black pre-commit hook ([docs](https://black.readthedocs.io/en/stable/integrations/source_version_control.html)).

Alternatively you can do something like [this](https://github.com/chrisimcevoy/pyoda-time/blob/705a0a39729e413b5e8e1b2b6da899a7eb6aad00/.pre-commit-config.yaml#L9-L10) at the `.pre-commit-config.yaml` file level

Reply:
My bad! Probably only really needed if we were to use something like pyupgrade anyway.